runestone/usageAssignment/__init__.py

```diff
@@ -57,10 +57,18 @@ def visit_ua_node(self,node):
         chapter_data = None
 
     s = ""
+    chapters_and_subchapters = {}
     if chapter_data and course_name:
-        for d in chapter_data:
+        for d in chapter_data: # Set up Chapter-Subchs dictionary
             ch_name, sub_chs = d['ch'], d['sub_chs']
-            s += '<div class="panel-heading">'
+            if d['ch'] not in chapters_and_subchapters:
+                chapters_and_subchapters[d['ch']] = [x for x in d['sub_chs']]
```

Comment:
Heh, yes. this is a simplified version of a more complicated thing that didn't get simplified enough.